### ISSUE TITLE
Dead loop when using pisa.CreatePDF() and image link

### ISSUE THREAD

**Issue by alispa** [xhtml2pdf/xhtml2pdf#596]
I am trying to create a pdf using `pisa.CreatePDF()` and a simple html just for testing:

```
<!DOCTYPE html>
<html lang="bs">
  <head>
    <meta charset="UTF-8" />
    <title></title>
  </head>

  <body>
    <h1>works</h1>

    <img src="https://cars.usnews.com/static/images/Auto/custom/14737/2022_Acura_ILX_1.jpg" alt="" />
    
  </body>
</html>
```

I run my python file in the terminal, it creates a new pdf file (ex. test.pdf) but the terminal never finishes, and the file stays open until I kill the terminal.

**The problem is the link, when I use other image links it works perfect.**

**Comment by luisza** [comment]
Yes, thanks, I commit a fix in master, but I need more tests to know if something else doesn't broke.

**Comment by luisza** [comment]
Hi, I checked what happen with this issue, but I don't know why the server is not answer the request (maybe it's required some https parameters).  But now by default we have a timeout so no more death loop in code.

**Comment by luisza** [comment]
Fixed on 0.2.6.
